confluent_server/confluent/messages.py

```diff
@@ -326,7 +326,7 @@ def get_input_message(path, operation, inputdata, nodes=None, multinode=False):
     elif path == ['events', 'hardware', 'decode']:
         return InputAlertData(path, inputdata, nodes)
     elif (path[:3] == ['configuration', 'management_controller', 'users'] and
-            operation != 'retrieve' and operation != 'delete'):
+      operation != 'retrieve' and operation != 'delete' and path[-1] != 'all'):
```

Comment:
I'm concerned about the indentation on this line.  Might need to be moved back to the previous indentation level.
Feel free to change the two  != to operation not in ('retrieve, 'delete')